src/api/track.ts

```diff
@@ -11,6 +7,8 @@ import Navigator from '../navigator';
 import Session from '../session';
 import Storage from '../storage';
 import TripsAPI from './trips';
+import { signJWT } from '../util/jwt'
+import { ping } from '../util/ping'
```

Comment:
Could maybe call this file `util/network` or just `util/net`

Reply:
Can update